script.js

```diff
@@ -0,0 +1,39 @@
+function calculate() {
+    var operationDropDown = document.getElementById("operation")
+    var firstNumElement = document.getElementById("firstNumber")
+    var secondNumElement = document.getElementById("secondNumber")
+    var firstNumber = parseInt(firstNumElement.value)
+    var secondNumber = parseInt(secondNumElement.value)
+    var answer = null
+
+
+
+    if (operationDropDown.value === "addition") {
+        answer = firstNumber + secondNumber
+        alert("Your answer is " + answer)
+
+    }
+    else if (operationDropDown.value === "subtraction") {
+        answer = firstNumber - secondNumber
+        alert("Your answer is " + answer)
+
+    }
+    else if (operationDropDown.value === "multiplication") {
+        answer = firstNumber * secondNumber
+        alert("Your answer is " + answer)
+
+    }
+    else if (operationDropDown.value === "division") {
+        answer = firstNumber / secondNumber
+        alert("Your answer is " + answer)
+
+    }
+    else {
+        alert("Please choose an operation.")
+
+    }
```

Comment:
A `switch` would have been cleaner here. Remember when we have many if/else tied together like this, `switch` is our friend.

```JavaScript
switch(operationDropDown.value) {
  case 'addition':
    answer = firstNumber + secondNumber
    alert('Your answer is ' + answer)
    break
  case 'subtraction':
    answer = firstNumber - secondNumber
    alert('Your answer is ' + answer)
    break
  case 'multiplication'
    answer = firstNumber * secondNumber
    alert('Your answer is ' + answer)
    break
  case 'division':
    answer = firstNumber / secondNumber
    alert('Your answer is ' + answer)
    break
  default:
    alert('Please choose an operation.')
}
```